src/Development/Shake/Command.hs

```diff
@@ -207,8 +207,9 @@ fsatraceFiles file = do
     xs <- parseFSAT <$> readFileUTF8 file
     let reader (FSATRead x) = Just x; reader _ = Nothing
         writer (FSATWrite x) = Just x; writer (FSATMove x y) = Just x; writer _ = Nothing
-    frs <- liftIO $ filterM doesFileExist $ nubOrd $ map normalise $ mapMaybe reader xs
-    fws <- liftIO $ filterM doesFileExist $ nubOrd $ map normalise $ mapMaybe writer xs
+        existing f = liftIO . filterM doesFileExist . nubOrd . mapMaybe f
```

Comment:
Are we talking about the `normalise` removal in this line? The purpose at this point is to make the comparison with the paths in `shakeLintInside` robust, but it seems cheaper to normalise just the `shakeLintInside` list (probably 1 or 2 entries) than the lists of accesses.